pkg/drivers/kic/oci/network_create.go

```diff
@@ -124,6 +126,15 @@ func dockerNetworkInspect(name string) (*net.IPNet, net.IP, error) {
 	return subnet, gateway, nil
 }
 
+func logDockerNetworkInspect(name string) {
+	cmd := exec.Command(Docker, "network", "inspect", name)
+	rr, err := runCmd(cmd)
```

Comment:
consider also running this debug func LogContainerDebug
https://github.com/medyagh/minikube/blob/e1615c81545be00876e76504ca1075ac9bfaba48/pkg/drivers/kic/oci/errors.go#L67

Reply:
done